dashboard/app/helpers/aichat_asset_helper.rb

```diff
@@ -8,8 +8,11 @@ def self.get_asset_data_uri(asset, channel_id, level_name)
     source = asset["source"]
     result = fetch_asset(filename, source, channel_id, level_name)
 
-    # TODO - error cases
-    return nil unless result && result[:status] == 'FOUND'
+    unless result && result[:status] == 'FOUND'
+      raise StandardError.new(
+        "Error fetching asset #{{**asset, channel_id: channel_id, level_name: level_name, **result}}"
```

Comment:
I was wondering if there was a more standard way to include these details than just in the error message string, but I don't see anything 😞